tests/cli/test_report.py

```diff
@@ -232,3 +232,25 @@ class args:
 
     shutil.move(backupfile, BUILD_REPORT)
     assert BUILD_REPORT
+
+
+@pytest.mark.cli
+def test_report_limited_rows():
+    class args:
+        filter = None
+        format = None
+        start = None
+        end = None
+        failure = None
+        latest = False
+        oldest = False
+        report_file = None
+        pager = None
+        terse = None
+        no_header = None
+        report_subcommand = None
+        helpfilter = False
+        helpformat = False
+        count = 5
+
+    report_cmd(args)
```

Comment:
Can you add another test with terse mode so just create another invocation to `args` . Alternately we may want to rewrite this test to directly call `Report` class directly. Either way works

Reply:
<img width="1425" alt="Screen Shot 2022-06-30 at 1 24 12 PM" src="https://user-images.githubusercontent.com/22202096/176739766-e1dc91ed-105f-423d-bdc2-9670bb4a8080.png">